src/main/java/net/mcreator/ui/datapack/recipe/BrewingRecipeMaker.java

```diff
@@ -36,15 +36,15 @@
 	public MCItemHolder cb2;
 	public MCItemHolder cb3;
 
-	public BrewingRecipeMaker(MCreator mcreator, MCItem.ListProvider itemsWithTags, MCItem.ListProvider items) {
+	public BrewingRecipeMaker(MCreator mcreator, MCItem.ListProvider itemsWithTags, MCItem.ListProvider itemsWithPotions) {
 		ImagePanel ip = new ImagePanel(UIRES.get("recipe.brewing").getImage());
 
 		ip.fitToImage();
 		ip.setLayout(null);
 
-		cb1 = new MCItemHolder(mcreator, itemsWithTags, true);
+		cb1 = new MCItemHolder(mcreator, itemsWithPotions, true);
```

Comment:
Shouldn't tags be supported here too, as itemsWithPotions will not provide tags?

cb3 is fine as result can't be tag, but cb1 could support them I think

Reply:
If you mean item tags like `#minecraft:small_flowers`, then those are already supported. Not sure if wildcards are needed for the input, since they default to a specific block anyway

Reply:
Maybe for older generators, where wildcards did not default to a specific block